src/heliumdb/utils.cpp

```diff
@@ -28,24 +28,27 @@ pickleDumps (PyObject* obj)
 PyObject*
 pickleLoads (const char* buf, size_t len)
 {
+    if (PICKLE_MODULE == NULL &&
+        (PICKLE_MODULE = PyImport_ImportModuleNoBlock ("pickle")) == NULL)
+        return NULL;
 #if PY_MAJOR_VERSION >= 3
     PyObject* pickedByteObj = PyBytes_FromStringAndSize (buf, len);
 #else
     PyObject* pickedByteObj = PyString_FromStringAndSize (buf, len);
 #endif
-
-    return PyObject_CallMethodObjArgs (PICKLE_MODULE,
+    PyObject* res = PyObject_CallMethodObjArgs (PICKLE_MODULE,
```

Comment:
Hey - you need to add the same code here for the loads method name, and then subsequently decref it at the end

C